src/documents/ChangeApplierAPI.md

```diff
@@ -60,14 +60,91 @@ A `<modelPathReference>` has the form:
-including the one-string compact syntax documented with [Invokers](Invokers.md#compact-format), and the use of [Priorities](Priorities.md). There are two extra features that are supported in
-model listener blocks that are not supported in standard listener declarations. These are the special context name `change`, and the ability to filter a change based on its _source_.
+A model listener declaration block has the same form and meaning as any of the record types supported by [Invokers](Invokers.md) and [Listeners](EventInjectionAndBoiling.md#listener-boiling) -
+including the one-string compact syntax documented with [Invokers](Invokers.md#compact-format), and the use of [Priorities](Priorities.md). Like standard event listener entries, model
+listeners can provide a `namespace` entry. Just one model listener with a particular namespace will be registered on a particular ChangeApplier. 
+
+A model listener declaration block includes three extra features beyond those found in ordinary event listeners. Firstly is the possibility of including a member [`path`](#the-path-entry-in-a-model-listener-declaration) of type `<modelPathReference>` (a `String` or `Object`), which can hold
+a more complex path specification for the the model listener match than can be encoded in the single string in a `<shortModelPathReference>`, secondly the ability to filter a change based on its [_source_](#source-tracking-and-filtering-in-model-listener-blocks), using the members
+`includeSource` and `excludeSource`, and finally the possibility that any IoC-resolved material in the listener declaration may match the special context name [`change`](#the-special-context-change-) which corresponds to the model change
+that the listener is reacting to. These entries are described in the linked sections below:
+
+#### The `path` entry in a model listener declaration ####
+
+The `path` entry fulfils the same basic function as the `<shortModelPathReference>` which may form the key of the listener declaration, but allows a richer set of path specifications to be used for 
+specifying which changes this model listener will respond to. Note that if the `path` entry is supplied, then the key of the listener will be interpreted as a _namespace_ for the listener rather than as
+a path specification.
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="3">Possible values for the <code>path</code> member of a model listener declaration (first two rows define <code>&lt;modelPathReference&gt;</code>)</th>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <th>Meaning</th>
+            <th>Examples</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>String</code></td>
+            <td>A <code>&lt;shortModelPathReference&gt;</code></td>
+            <td>
+                See examples in section [Model Path References](#model-path-references)
```

Comment:
The markdown syntax does not take effect when being placed in a html block. Need to use html instead, such as

`<a href="#model-path-references">Model Path References</a>`